skema/skema-rs/mathml/src/expression.rs

```diff
@@ -1,8 +1,17 @@
 use crate::ast::{
     Math, MathExpression,
-    MathExpression::{Mi, Mn, Mo, Mrow, Msub},
+    MathExpression::{Mfrac, Mi, Mn, Mo, Mrow, Msqrt, Msub},
     Operator,
 };
+use petgraph::dot::{Config, Dot};
+use petgraph::graph::Node;
+use petgraph::visit::NodeRef;
+use petgraph::{graph::NodeIndex, Graph};
+use std::cmp::Reverse;
+use std::collections::VecDeque;
+use std::ptr::null_mut;
+
+pub type MathMLGraph<'a> = Graph<String, String>;
```

Comment:
I don't think this graph has any trace of MathML left in it after you've done your transformations, right? Perhaps you should name it something different? Like `MathExpressionGraph`?

Reply:
Thanks! I updated the name as you suggested.